src/main/java/com/sucy/skill/dynamic/trigger/ChatTrigger.java

```diff
@@ -17,7 +17,7 @@ public class ChatTrigger implements Trigger<AsyncPlayerChatEvent> {
      */
     @Override
     public String getKey() {
-        return "CHAT";
+        return "CHATx";
```

Comment:
Any particular reason this needed to change?

Reply:
That is correct, I was simply testing something and I did not mean to submit it. I'm rather new to this so I didn't know if it was possible to take one thing back. Only the change for the new variable {playeruuid} is needed. I'm sorry for the confusion!

Reply:
You can make the change again and re-commit, or you can use `git revert` to undo the commit.